index.html

```diff
@@ -0,0 +1,144 @@
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <!-- <h1>Hello World, we are software developers!</h1> -->
+
+    <!-- form code starts here -->
+
+    <div class="container">
+      <header class="header">
+        <h1 id="title" class="text-center">freeCodeCamp Survey Form</h1>
+        <p id="description" class="description text-center">
+          Thank you for taking the time to help us improve the platform
+        </p>
+      </header>
+      <form id="survey-form">
+        <div>
+          <label id="name-label" for="name">Name</label>
+          <input
+            type="text"
+            name="name"
+            id="name"
+            placeholder="Enter your name"
+            required
+          />
+        </div>
+
+        <div >
+          <label id="email-label" for="email">Email</label>
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="Enter your Email"
+            required
+          />
+        </div>
+        <div >
+          <label id="number-label" for="number"
+            >Number</label
+          >
+          <input
+            type="number"
+            name="age"
+            id="number"
+            min="10"
+            max="99"
+            class="form-control"
+            placeholder="Age"
+          />
+        </div>
+        <div>
+          <select id="dropdown" required>
+            <option disabled selected value>Select</option>
+            <option value="First Option">First Option</option>
+            <option value="Second Option">Second Option</option>
+          </select>
+        </div>
+    
+        <div >
+            <input
+              name="sex"
+              value="male"
+              type="radio"
+            />
+        
+            <input
+              name="sex"
+              value="female"
+              type="radio"
+            />
+        
+        </div>
+    
+    
+        <div class="form-group">
+          <p>
+            You can select many
+          </p>
+
+          <input
+              name="first"
+              value="select first"
+              type="checkbox"
+            />
+
+            <label>
+              <input
+              name="second"
+              value="select second"
+              type="checkbox"
+            />
+              Select First
+            </label>
+            
+          <label
+            ><input
+              name="third"
+              value="select third"
+              type="checkbox"
+            />Select Third</label>
+        </div>
+    
+        <div>
+          <p>Add comments if you have</p>
+          <textarea
+          ></textarea>
+        </div>
+    
+        <div>
+          <button type="submit" id="submit">
+            Submit
+          </button>
+        </div>
+      </form>
+    </div>
+    
+    
+
+
+
+
```

Comment:
In [index.html](https://github.com/oluwatobi-fasasi/Code-Review/blob/code-review/index.html)

- [ ] **I suggest you close up block of spaces as it is not a good coding practice since it makes your code looks scattered.**